main.go

```diff
@@ -109,10 +133,13 @@ func sendFunnyEmail(w http.ResponseWriter, r *http.Request) {
 	json.Unmarshal(reqBody, &email)
 
 	// fetch dad joke
-	dadJoke := getDadJoke()
+	dadJoke, err := getDadJoke()
+	if err != nil {
+		w.WriteHeader(400)
```

Comment:
would you normally pass the status code from the fetch dad joke call here? is it available on err or would I need to return it as a third item in getDadJoke?

Reply:
you could depending on how the app handled errors. but usually you determine the status code for your app which could be based off the call but doesn't have to be.